src/one_dragon/base/screen/screen_area.py

```
@@ -7,19 +7,20 @@
-                 template_id: Optional[str] = '',
-                 template_sub_dir: Optional[str] = '',
-                 template_match_threshold: float = 0.7,
-                 pc_alt: bool = False,
-                 id_mark: bool = False,
-                 goto_list: Optional[list[str]] = None,
-                 color_range: Optional[list[list[int]]] = None,
-                 ):
+    def __init__(
+        self,
+        area_name: str = '',
+        pc_rect: Rect = Rect(0, 0, 0, 0),
```

Ruff (0.14.7)

Do not perform function call `Rect` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)